modules/serverless-rn-1-20-0.adoc

```diff
@@ -0,0 +1,57 @@
+[id="serverless-rn-1-20-0_{context}"]
+= Release Notes for Red Hat {ServerlessProductName} 1.20.0
```

Comment:
Confused about why this is in two different PRs - is this troubleshooting guide being pushed out for 1.21.0?